### PR TITLE
(update) new vertical responsive buttons

### LINKED ISSUE #133: Responsive crypto select buttons
The buttons on the select crypto page require updating to use the new vertical design, which uses the `<button>` tag instead of a table.
The woocommerce plugin can be used as a reference: https://github.com/blockonomics/woocommerce-plugin/pull/254
Instead of `class="woocommerce-button"`, [bootstrap classes](woocommerce-button) can be used as we are currently importing the [bootstrap prestashop ui kit](https://github.com/blockonomics/prestashop-plugin/blob/master/controllers/front/validation.php#L42)

### PR DESCRIPTION
**Pull request by thisisayush**
Fixes #133 

image

### CODE REVIEW

File: views/css/order.css [hunk 1]
```diff
@@ -192,35 +195,18 @@
...
-  margin-left: auto;
-  margin-right: auto;
-}
-
 .bnomics-select-options {
   cursor:pointer;
-  width:50%;
+  width:100%;
+  display: block;
+  height: 4.2em;
```
Comment (DarrenWestwood) [inline views/css/order.css:202]: More themes should be tested to ensure the new layout works with different themes.

For instance [this theme](https://templatetoaster.com/template/fhoto-studio-online-animals-photography-prestashop-theme) includes the following CSS
```
.btn:hover {
    clear: none;
    height: auto;
}
```
This leads to the size of the button changing on hover. We can consider using min-height instead.

Comment (thisisayush) [reply views/css/order.css:202]: I think we should remove height altogether. If the buttons are supposed to match the site's theme, they should match the inner paddings, and height as well.

Comment (DarrenWestwood) [reply views/css/order.css:202]: We can remove the height altogether, however this should not cause issues with the icons or vertical line on smaller sized buttons.

Comment (thisisayush) [reply views/css/order.css:202]: I tested it with 3 themes after removing the height attribute, the button is adjusting itself with the height required.
The vertical line is just a border so it'll never overflow.

Comment (DarrenWestwood) [reply views/css/order.css:202]: Okay, lets remove the height attribute, `<tr>`, and the commented out border-radius/text-transform.

Comment (thisisayush) [reply views/css/order.css:202]: Okay

**Comment by thisisayush** [comment]
PS Validator Passed ✅

**Review by DarrenWestwood — APPROVED** [review]

**Pull request merged**